tests/js/spec/views/eventsV2/miniGraph.spec.tsx

```diff
@@ -59,6 +59,14 @@ describe('EventsV2 > MiniGraph', function () {
     });
     // @ts-expect-error
     eventView = EventView.fromSavedQueryOrLocation(undefined, location);
+
+    // @ts-expect-error
+    MockApiClient.clearMockResponses();
+    // @ts-expect-error
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/events-stats/',
+      statusCode: 200,
+    });
```

Comment:
i'm so confused lol how did these tests work without the mock @edwardgou-sentry

Reply:
I think it's because previously we didn't have an `await tick()` in any of the test cases (since it wasn't necessary).
Previous enzyme behaviour probably evaluated all the conditions before firing off and receiving a mocked response, so the tests passed before it could complain about mocks.

Adding the `await tick()` will cause the tests to wait for the `eventsRequest` response which would result in errors if there is no mock.